sdk/storage/Azure.Storage.Blobs/AzureStorageNetMigrationV12.md

```diff
@@ -247,24 +239,38 @@ async static Task CreateStoredAccessPolicyAsync(string containerName)
 }
 ```
 
-### Client hierarchy
+### Client structure
 
-In the interest of simplifying the API surface we've made a three top level clients that can be used to interact with a majority of your resources: `BlobServiceClient`, `BlobContainerClient`, and `BlobClient`.
+The legacy SDK was structured like an object model. There were container and blob objects that held state and required the user to manually call their update methods. But blob contents were not a part of this state and had to be uploaded/downloaded whenever they were to be interacted with. This became increasingly confusing over time, and increasingly susceptible to thread safety issues.
```

Comment:
[Object model](https://en.wikipedia.org/wiki/Object_model) isn't synonym for state-full. Both `BlobClient` and `CloudBlob` together with surround data models are object oriented representations of blobs. The difference is that one is state-full and the other one is state-less and this is the terminology that should be used to describe the difference.